features/cellular/framework/device/CellularStateMachine.cpp

```diff
@@ -373,6 +374,22 @@ void CellularStateMachine::state_device_ready()
             }
         }
     }
+
+#if MBED_CONF_MBED_TRACE_ENABLE
+    CellularInformation *info = _cellularDevice.open_information();
```

Comment:
Should this be done only when moving to next state (STATE_SIM_PIN)? We might be restarting/powering or failing to communicate with the modem.

Reply:
Yes, this should probably be just before `if (device_ready())` at line 364.